### PR TITLE
WOC 6 Cards

### PR DESCRIPTION
**Pull request by kevlahnota**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/court_of_ardenvale.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Court of Ardenvale
+ManaCost:2 W W
+Types:Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMonarch | TriggerDescription$ When CARDNAME enters the battlefield, you become the monarch.
+SVar:TrigMonarch:DB$ BecomeMonarch | Defined$ You
+T:Mode$ Phase | Phase$ Upkeep | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigBranch | TriggerDescription$ At the beginning of your upkeep, return target permanent card with mana value 3 or less from your graveyard to your hand. If you're the monarch, return that permanent card to the battlefield instead.
+SVar:TrigBranch:DB$ Branch | BranchConditionSVar$ X | BranchConditionSVarCompare$ GE1 | TrueSubAbility$ TrigPlay | FalseSubAbility$ TrigReturn
+SVar:TrigReturn:DB$ ChangeZone | ValidTgts$ Permanent.YouCtrl+cmcLE3 | TgtPrompt$ Select target permanent card with mana value 3 in your graveyard | Origin$ Graveyard | Destination$ Hand
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/court_of_ardenvale.txt:8]: engine doesn't see these Tgts on condition subs
you need to move it to Branch effect
(AI might not like it)

Comment (kevlahnota) [reply forge-gui/res/cardsfolder/upcoming/court_of_ardenvale.txt:8]: is there an existing card/s with example like those you mentioned?

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/court_of_ardenvale.txt:8]: target only works on the main sub ability chain

additional sub abilities can't have target

to make it work, you need a Pump before the Branch / or Target on Branch (depending on AI?)

Comment (kevlahnota) [reply forge-gui/res/cardsfolder/upcoming/court_of_ardenvale.txt]: @Hanmac @tool4ever  I update it works on player but not on AI?

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/court_of_ardenvale.txt]: might need AI logic, but that can be done later

**Review by tool4ever — DISMISSED** [review]

File: forge-gui/res/cardsfolder/upcoming/court_of_ardenvale.txt (hunk 1 above)
Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/court_of_ardenvale.txt]: @kevlahnota do you want to try to write it as condition instead?

Comment (kevlahnota) [reply forge-gui/res/cardsfolder/upcoming/court_of_ardenvale.txt]: > @kevlahnota do you want to try to write it as condition instead?

done seems to work fine

**Review by tool4ever — APPROVED** [review]

**Pull request merged**